maxminddb/file.py

```diff
@@ -21,7 +21,7 @@ def __getitem__(self, key):
         if isinstance(key, slice):
             return self._read(key.stop - key.start, key.start)
         if isinstance(key, int):
-            return self._read(1, key)
+            return self._read(1, key)[0]
```

Comment:
What was the difference?

Reply:
In Python 3, the buffers are `bytes`, which is a sequence of bytes. When you index a `bytes`, you get back an integer in the range 0 to 255, but the file read was returning a `bytes` of length 1. This different did not matter previously, but with the other changes, it did.